pandas/tests/series/test_reductions.py

```diff
@@ -51,29 +53,32 @@ def test_td64_sum_empty(skipna):
     assert result == pd.Timedelta(0)
 
 
-def test_td64_summation_overflow():
-    # GH#9442
-    ser = Series(pd.date_range("20130101", periods=100000, freq="H"))
-    ser[0] += pd.Timedelta("1s 1ms")
+@given(
+    st.integers(
+        min_value=0,
+        max_value=10 ** (np.finfo(np.float64).precision),
```

Comment:
any reason not to write `10**15` explicitly?

Reply:
Nope, will do.